docs-src/workshop-fomo.scrbl

```diff
@@ -0,0 +1,243 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@(define TAG "workshop-fomo")
+@title[#:version reach-vers #:tag TAG]{Workshop: FOMO}
```

Comment:
You should expand out FOMO to "Fear of missing out" and maybe reference the original FOMO3D game (although their site is down)